### DISCUSSION TITLE
how-to-push-an-empty-commit-in-git

### DISCUSSION THREAD

**Discussion by giscus** [thomasnotfound/how.wtf#88] [Blog Posts]
# how-to-push-an-empty-commit-in-git

How to push without making changes in git.

https://how.wtf/how-to-push-an-empty-commit-in-git.html

**Comment by ra2dev** [comment]
Interesting,
It is useful for conventional commits - if smb forgot to add convential commit that will be used in auto-generated changelog, you can make empty commit with proper text, instead of doing dump changes (https://github.com/lerna/lerna-changelog)

**Reply by thomasgtaylor** [reply]
Thank you for your comment. Agreed! It serves well for that purpose.
